### ISSUE TITLE
Questions about flashing SXJ02ZM

### ISSUE THREAD

**Issue by badigit** [anmaped/openfang#61]
Is [here](https://github.com/anmaped/openfang/blob/master/doc/SXJ02ZM/SXJ02ZM_instructions.md) a complete manual?

After reprogramming, do I should to solder pin 8 back? Or not?
It is neccesary remove the 2 frontpanel LED connectors before flash?

**Comment by mgx0** [comment]
Yes you need to solder it back otherwise the eeprom will not work
